### PR TITLE
Adding Issue Template

### PR DESCRIPTION
**Pull request by abhimeda**
Hello, this PR is introducing a new issue template for ROCm public repositories to make it easier for users to submit the details for issues and bugs. Problems pertaining to documentation or non-bug related, users have the ability to open a blank issue template to submit their report. 

Thank you,
Community Support Team (ML SW SDK)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue_report.yml [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  attributes:
+    label: ROCm Component
+    description: (Optional) If this issue relates to a specific ROCm component, it can be mentioned here.
+    multiple: true
+    options:
+        - Other
+        - AMD Common Language Runtime
+        - AMD MIGraphX
+        - AMD System Management Interface
+        -  amdgpu KCL/autoconf
```
Comment (samjwu) [inline .github/ISSUE_TEMPLATE/issue_report.yml]: small typo: an extra space

**Review by samjwu — CHANGES_REQUESTED** [review]
Can you add MI300 as an option?

**Review by illsilin — DISMISSED** [review]

**Comment by illsilin** [comment]
I think the list of architectures is probably not very suitable for CK. 
We only support MI100/200/300, and Navi3x. 
So we need to add the MI300 and Navi3x, and delete the rest.

**Comment by illsilin** [comment]
I've added the MI300 architectures and also the rocm 5.6.1 and 6.0.0 versions, since 6.0 is about to be released any day now.

**Review by illsilin — APPROVED** [review]

**Review by samjwu — APPROVED** [review]

**Pull request merged**